Jenkinsfile

```
@@ -26,35 +26,36 @@ node('docker&&linux') {
 
     timestamps {
         stage('Generate Plugin Data') {
-            docker.image('maven').inside {
-              sh 'mvn -PgeneratePluginData'
-            }
+          infra.runWithMaven(
+            'mvn --no-transfer-progress -PgeneratePluginData',
+            /*jdk*/ '11',
```

```suggestion
            /*jdk*/ '17',
```
This step works fine with jdk17: https://ci.jenkins.io/job/Infra/job/plugin-site-api/job/PR-120/7/console